wallaby.js

```diff
@@ -0,0 +1,38 @@
+module.exports = (wallaby) => ({
+  files: [
+    "src/**/*.ts",
+    "test/support/*.ts",
+    "locale/lang.*.json",
+    "package.json",
+    "package.nls.json",
+    "package.nls.template.json",
+    "icons/*.*"
+  ],
+  tests: [
+    "test/**/*.test.ts"
+  ],
+  filesWithNoCoverageCalculated: [
+  ],
+  preprocessors: {
+    '**/*.json': (file, done) => {
+      done(file.rename(`../${file.path}`).content)
+    },
+    "icons/*.*": (file, done) => {
+      done(file.rename(`../${file.path}`).content)
+    }
+  },
+  hints: {
+    ignoreCoverage: /\/* wallaby ignore next\/*/
+  },
+  testFramework: "mocha",
+  env: {
+    type: "node",
+    runner: `${require('os').homedir()}/.nvm/versions/node/v7.9.0/bin/node`
```

Comment:
I already merged this and forgot to tell you that this isn't working in Windows environments. My nvm gets installed in `userDir`\AppData\Roaming\nvm

Reply:
Honestly I didn't test it in Windows as I do all development in Linux lately. I'll have a look when I find some time.

Reply:
I've submitted a PR removing the runner line. I think we can safely remove it. Devs using wallaby (namely you, @jens1o and me) already know which is the node version supported so we are responsible of selecting the approppriate node version in our machines.

Reply:
tbh I didn't set it up yet :D